testing/tests/simple.rs

```diff
@@ -490,7 +490,7 @@ fn test_num_literals() {
     let template = NumLiterals;
     assert_eq!(
         template.render().unwrap(),
-        "[90, -90, 90, 2, 56, 240, 10.5, 10.5, 100000000000, 105000000000]\n1\n12",
+        "[90, -90, 90, 2, 56, 240, 10.5, 10.5, 100000000000.0, 105000000000.0]\n1\n12",
```

Comment:
ryu adds a trailing `.0` for integral floats. IMO ryu gets it more right than rust's defaults, but we could also strip a trailing `.0` to remove the difference.

Reply:
I don't think it matters much. If people want to change it, they can always using the `format` filter.

Reply:
Hum... Thinking some more about it, people will expect us to use the Rust implementation to get the same output. I think it's problematic that `ryu` doesn't always display the same thing.

Here what I suggest: I talked with Mara and she said that we could use `ryu`'s algorithm directly in `std` to improve performance. I think it's a better way forward rather than having our own display, the performance difference is not big enough to be worth imo.

Reply:
The performance is much more impacted by the removal of the `&dyn Write`, the speed-up through `ryu` is minuscule in comparison. It's just the first implementation that came to mind.

But I think it's seldom that people will want to print floats, and then most likely with explicit formatting (`price|fmt(".02")`), so it won't matter much if I remove this specialization.